translator/totomlconfig/tomlConfigTemplate/tomlConfig.go

```diff
@@ -33,7 +33,7 @@ type (
 		Cpu               []cpuConfig
 		Disk              []diskConfig
 		DiskIo            []diskioConfig
-		Eththool          []ethtoolConfig
+		Ethtool           []ethtoolConfig
```

Comment:
I'm surprised that this has been broken for so long. Is there a unit test that covers this?

Reply:
TestAdvancedConfig returned nil when trying to do struct translation. That is how this was found. Now the struct translation returns the proper value.

Reply:
But the git blame shows this line is 5 months old. What you're saying is our unit test hasn't caught this in >5 months, and making a change here doesn't impact the unit test?

Reply:
Yes